src/services/rolesServices.js

```diff
@@ -0,0 +1,52 @@
+import models from '../database/models';
+
+const { User, Role } = models;
+const isSuperAdmin = async (email, userId) => {
+  const user = await User.findOne({
+    where: { email: `${email}`, id: userId },
```

Comment:
why are you using string literals here? Email will always be a string. no need for the ${email}